index.js

```diff
@@ -55,101 +55,43 @@ try {
         GITHUB_ACTION: process.env.GITHUB_ACTION,
         GITHUB_REF: process.env.GITHUB_REF,
         GITHUB_REPOSITORY: process.env.GITHUB_REPOSITORY,
-        GITHUB_SHA: process.env.GITHUB_SHA
+        GITHUB_SHA: process.env.GITHUB_SHA,
+        GITHUB_HEAD_REF: process.env.GITHUB_HEAD_REF
```

Comment:
Any way we can check to see if it exists first? Wouldn't this throw an error according to these docs: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-environment-variables#default-environment-variables

Reply:
error? there will not be a error its just going to result undefined (same for other env variables)

Reply:
i changed it to
```js
GITHUB_HEAD_REF: process.env.GITHUB_HEAD_REF || ''
```